spec/helpers/medications_helper_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+describe MedicationsHelper do
+  let(:current_user) { create(:user) }
+  let(:medication) { create(:medication, user: current_user) }
+
+  before(:each) do
+    @medication = medication
+  end
+
+  def get_field(field_name)
+    @fields.find {|f| f[:id] == field_name}
+  end
+
+  describe 'fields' do
+    before(:each) do
+      @fields = common_fields
+    end
+
+    it 'returns common medication fields' do
```

Comment:
So you'll want to test the direct output of new_medication_props and edit_medication_props rather than common_fields which is a private method.